galley/pkg/config/analysis/msg/messages.yaml

```diff
@@ -209,3 +209,16 @@ messages:
     args:
       - name: deployment
         type: string
+
+  - name: "PortNameIsNotUnderNamingConvention"
+    code: IST0118
+    level: Info
```

Comment:
Is info shown to users by default?

Reply:
I am not familiar with the istioctl analyzer. I think info is also shown to users by default but maybe not highlighted.

Reply:
Info is shown by default, but won't cause the command to fail with a nonzero exit code.